docs/root/configuration/other_features/dlb.rst

```diff
@@ -33,7 +33,62 @@ If enabled, the Dlb connection balancer will:
 When new connections come, one worker thread will accept it and send it to Dlb hardware. Dlb hardware
 does balancing then trigger one worker thread to receive via libevent.
 
-Installing and using Dlb
-------------------------
+Installing Dlb
+--------------
 
-For information on how to build/install and use libdlb see `the getting started guide <https://downloadmirror.intel.com/727424/DLB_Driver_User_Guide.pdf>`_.
+Download DLB driver release tarball from `dlb website <https://www.intel.com/content/www/us/en/download/686372/intel-dynamic-load-balancer.html>`_, then install it refer to `the getting started guide <https://downloadmirror.intel.com/727424/DLB_Driver_User_Guide.pdf>`_.
+
+
+Using Dlb
+---------
+
+Create a config file named `dlb-config.yaml` to make Envoy listen 10000 port as proxy, the upstream server listens 12000 port.
+
+.. literalinclude:: _include/dlb_example_config.yaml
+ :language: yaml
+
+Run the upstream service:
+
+.. code-block:: bash
+
+ $ docker run -d -p 12000:80 nginx
+
+Run Envoy with dlb enabled:
+
+.. code-block:: bash
+
+ $ ./envoy --concurrency 2 -c dlb-config.yaml
```

Comment:
```suggestion
 $ ./envoy --concurrency 2 -c dlb_example_config.yaml
```

rather keep the name of the example file so it can be downloaded and run without any change